components/map.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Mapsearch from './mapsearch.jsx';
+
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+  }
+
+  componentDidMount() {
+    const theMap = this.refs.map;
+    const mapOptions = {
+      center: {lat: 37.773972,
+      lng: -122.431297},
+      zoom: 12,
+      minZoom: 3
+    };
+
+    this.initializeMap();
+  }
+
+  initializeMap() {
+    var chicago = {lat: 41.850033, lng: -87.6500523};
+
+    var INITIAL_LOCATION = {
+      address: 'Chicago, Illinois',
```

Comment:
The initial location seems to be a mix of SF and Chicago. Why is line 24 showing different lat/lng coordinates?